examples/intro-original-language-with-dub-language-and-adaptation.xml

```diff
@@ -0,0 +1,24 @@
+<tt xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:daptm="http://www.w3.org/ns/ttml/profile/dapt#metadata"
+    xml:lang="en"
+    ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/dapt"
+    daptm:scriptType="DUBBING_PRE_RECORDING">
+  <head>
+    <metadata>
+      <ttm:agent type="character" xml:id="character_1">
+        <ttm:name type="alias">ASSANE</ttm:name>
+      </ttm:agent>
+    </metadata>
+  </head>
+  <body>
+    <div begin="10s" end="13s" ttm:agent="character_1">
+      <p xml:lang="fr" daptm:langSrc="original">
+        <span>Et c'est grâce à ça qu'on va devenir riches.</span>
+      </p>
+      <p daptm:langSrc="translation">
```

Comment:
I wonder if it would be helpful to add some metadata here providing guidance, e.g. a pronunciation note, or an emotion note, in a `<ttm:desc>`?

Reply:
I'd prefer keeping this example simple. We have examples with these extra features in the text.